css/welcome.css

```diff
@@ -313,6 +294,35 @@ main {
     height: auto;
     transform: none;
     padding: 0.75em 0.5em;
+
+    background: #bfdaf5;
+    border-radius: 5px;
+    border: 0.5vmin solid var(--accent-color);
+
+    box-sizing: border-box;
+
+    font-weight: bold;
+    text-align: center;
+    text-decoration: none;
+    color: #0e00c3;
+    font-family: "Rowdies", sans-serif;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  #welcome-nav a.special-nav-card {
+    /* border gradient only seems to work when adding it to the bg property */
+    border-color: transparent;
+    background: linear-gradient(45deg, #a7daf5, #ffffff, #a7daf5) padding-box,
```

Comment:
The glow behind the card seems to not be perfectly aligned with the new images

Reply:
Removed in the lastest commit!